index.html

```diff
@@ -0,0 +1,1084 @@
+                               </a>
+                            </div>
+                        </div>
+                        <div class="col-md-4 left-side d-flex order-md-3 pb-3 pt-3">
+                            <div class="left-side">
+                                    <a href="#" class="login float-left pt-1 pb-1">SIGN IN</a>
+                                    <a href="#" class="button float-right ml-3 pr-2 pt-1 pb-1 pl-2">SUBSCRIBE
+                                        <i class="fas fa-angle-right mr-2 ml-2"></i>
+                                    </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-12 mb-1">
+                            <nav class="header-bottom">
+                                
+                                <ul class="page-nav">
+                                    <li><a href="#">U.S.</a></li>
+                                    <li><a href="#">World</a></li>
+                                    <li><a href="#">Business</a></li>
+                                    <li><a href="#">Tech & Science</a></li>
+                                    <li><a href="#">Culture</a></li>
+                                    <li><a href="#">Sports</a></li>
+                                    <li><a href="#">Health</a></li>
+                                    <li><a href="#">Opinion</a></li>
+                                    <li><a href="#">Vantage</a></li>
+                                    <li class="label">Search</li>
+                                    <li class="search-icon">Search </li>
+                                </ul>
+                            </nav>
+                    </div>
+                </div>
+    </header>
+   
+   <div class="main container-fluid">
+       <div class="row header-main">
+            <div class="col-md-3 ml-2 momentum">
+                <a class="topic" href="#">FEATURED STORIES
+                </a>
+                <div class="side-image">
+                <a href="#"><img src="../NewsWeek/Img/newsweek-momentum-awards-2019.png" class="img-responsive" alt="Responsive image"></a>
+                <div class="bottom-image-red"><strong>ANALYSIS</strong></div>
+                </div>  
+                <h1>
+                    <a href="#">The Newsweek Momentum Awards
+                    </a>
+                </h1>
+                <p class="text-left mr-5">The Newsweek Momentum Awards will recognize five individuals around the globe, as well as one city, which will be chosen as the "smartest city in the world.</p>
+                
+                <div class="side-image">
+                <a href="#"><img src="../NewsWeek/Img/us-embassy-moscow-russia-protest-map.jpg" class="img-responsive" alt="Responsive image"></a>
```

Comment:
Do it for all of them. Right now images are not showing if you access to the [live demo.](https://rawcdn.githack.com/maya88en/NewsWeek/f9233c619fb750a37de632d24b735ea0d408b684/index.html)